### ISSUE TITLE
Training Settiings

### ISSUE THREAD

**Issue by cjd-hub** [Kami-code/dexart-release#23]
Thank you for your great contribution!The code I use for reinforcement learning training directly causes my computer to freeze, even when I adjust the batch_size to 5. May I ask how many environments are used during training? And what are the requirements for the computer's graphics card for training?

**Comment by Kami-code** [comment]
Hi @cjd-hub . Thanks for using our benchmark. Usually, each training seed takes about 48 hours for Faucet, 72 h. for Bucket, 15 h. for Laptop, and 15 h. for Toilet on an NVIDIA RTX 2080Ti GPU when you use --n 100 --workers 10 --iter 5000 --bs 500 in our training example.

The details of our machine can be found [here](https://github.com/Kami-code/dexart-release/issues/1#issuecomment-1611352302). The training details can be found [here](https://github.com/Kami-code/dexart-release/issues/1#issuecomment-1557198625).
